charts/agent-control/values.yaml

```diff
@@ -8,19 +8,32 @@ toolkitImage:
   registry:
   repository: newrelic/newrelic-agent-control-cli
   # @default It defaults to `appVersion` in `Chart.yaml`.
-  tag: "0.40.0"
+  tag: "0.41.0"
   pullPolicy: IfNotPresent
   # -- The secrets that are needed to pull images from a custom registry.
   pullSecrets: []
 
+installationJob:
+  # -- The repository URL from where the `agent-control-deployment` chart will be installed.
+  chartRepositoryUrl: https://helm-charts.newrelic.com
```

Comment:
thanks! Notice that needs is a small breaking change for our e2e tests, we need to fix that as well when we point to master